languages/english.json

```diff
@@ -0,0 +1,23 @@
+{
+  "translation": {
+    "CreateTaskButtonTitle": "Enter new task title:",
```

Comment:
For json file better use separating to file/screen. f.e. 
"HomeScreen" : {
    "ok": "ok"
}